pygmt/datasets/load_remote_dataset.py

```diff
@@ -87,7 +87,7 @@ class GMTRemoteDataset(NamedTuple):
     "earth_free_air_anomaly": GMTRemoteDataset(
```

Comment:
Also need to update the failing tests.

Reply:
The tests are updated in commit https://github.com/GenericMappingTools/pygmt/pull/2758/commits/ec6f36cddb0c6476973927c376f0874f8e9759bb.